app/src/main/res/layout/fragment_car.xml

```diff
@@ -7,11 +7,24 @@
     android:layout_height="match_parent"
     tools:context=".ui.cars.CarFragment">
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/addCar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:clickable="true"
+        android:contentDescription="Add new car"
+        android:src="@android:drawable/ic_input_add"
+        app:backgroundTint="@color/purple_500"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:tint="@color/white" />
+
```

Comment:
I added floating buttons to the fragments instead of adding a button to the action bar because it is a lot simpler.